jetty-core/jetty-http-spi/src/main/java/org/eclipse/jetty/http/spi/JettyHttpServer.java

```diff
@@ -39,7 +42,7 @@
 /**
  * Jetty implementation of {@link com.sun.net.httpserver.HttpServer}.
```

Comment:
javadoc link need to be updated

```suggestion
 * Jetty implementation of {@link com.sun.net.httpserver.HttpsServer}.
```

Reply:
HttpsServer itself extends HttpServer, so it's still technically correct. Even so, I have updated it